pkg/controller/services.go

```diff
@@ -397,11 +422,16 @@ func (cont *AciController) updateServiceDeviceInstance(key string,
 		nodes = append(nodes, node)
 	}
 	sort.Strings(nodes)
-
 	name := cont.aciNameForKey("svc", key)
+	var conScope string
+	scope_val, ok := service.ObjectMeta.Annotations[metadata.ServiceScopeAnnotation]
+	if ok {
+		conScope = scope_val
```

Comment:
I assume ok could also mean its empty "" ?

Reply:
If it's empty, we default to "context", but empty string is still acceptable